README.md

```diff
@@ -15,26 +15,29 @@ $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0
 
 # Answer:
 
-First inequality:
+If we cannot assume a particular T(n) or c, then we must prove that 
 
-$c_1 \cdot log_{2} n \leq log_{5} n$
+$O(log_{2} n) \subseteq O(log_{5} n)$ and $O(log_{5} n) \subseteq O(log_{2} n)$
 
-$log_{5} n = log_{2} n / log_{2} 5$ 
+T(n) $\leq c \cdot log_{5} n$
 
-$c_1 = 1 / log_{2} 5$
+$log_{5} n$ = $\frac {log_{2} n} {log_{2} 5} $
 
-$1 / log_{2} 5 \cdot log_{2} n \leq log_{5} n$
+T(n) $\leq c \cdot \frac {log_{2} n} {log_{2} 5}$
 
-$log_{2} n / log_{2} 5 \leq log_{5} n$
+T(n) $\leq \frac {c \cdot log_{2} n} {log_{2} 5}$
 
-Second inequality:
+$log_{2} 5$ is combined with c because they are both constants
```

Comment:
If you're multiplying something other than 1 with $c$, you're not getting the same $c$.